integration-tests/src/test/java/org/alfresco/core/SitesApiServiceIntegrationTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021-2021 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.alfresco.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.alfresco.core.handler.NodesApiClient;
+import org.alfresco.core.handler.SitesApiClient;
+import org.alfresco.core.model.SiteBodyCreate;
+import org.alfresco.core.model.SiteEntry;
+import org.alfresco.core.model.SiteMemberEntry;
+import org.alfresco.core.model.SiteMembershipBodyCreate;
+import org.alfresco.core.model.SiteMembershipBodyCreate.RoleEnum;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.shaded.org.apache.commons.lang.RandomStringUtils;
+
+@Testcontainers
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SitesApiServiceIntegrationTest {
+
+    @Container
+    public static DockerComposeContainer compose = new DockerComposeContainer(new File("src/test/resources/docker-compose.yml"))
```

Comment:
These literal values like: 
* Name of the service
* Port
* Duration in seconds
* Poll delay
Would it be OK to promote them to constant or config properties?